.codeclimate.yml

```diff
@@ -1,7 +1,12 @@
 ---
 exclude_paths:
 - ".git/"
+- "**.html"
+- "**.json"
+- "**.xml"
+- "**.yaml"
 - "**.yml"
+- "**.gif"
```

Comment:
Can we alphabetically sort this list, even if only within groups like everything starting with `**.`?